### PR TITLE
fix typos in EIP 1

### PR DESCRIPTION
**Pull request by guanqun**

### CODE REVIEW

File: EIPS/eip-1.md [hunk 1]
```diff
@@ -42,13 +42,13 @@ Each EIP must have a champion - someone who writes the EIP using the style and f
...
 
 Standards Track EIPs consist of three parts, a design document, implementation, and finally if warranted an update to the [formal specification]. The EIP should be reviewed and accepted before an implementation is begun, unless an implementation will aid people in studying the EIP. Standards Track EIPs must be implemented in at least three viable Ethereum clients before it can be considered Final.
 
-For an EIP to be accepted it must meet certain minimum criteria. It must be a clear and complete description of the proposed enhancement. The enhancement must represent a net improvement. The proposed implementation, if applicable, must be solid and must not complicate the protocol unduly.
+For a EIP to be accepted it must meet certain minimum criteria. It must be a clear and complete description of the proposed enhancement. The enhancement must represent a net improvement. The proposed implementation, if applicable, must be solid and must not complicate the protocol unduly.
```
Comment (Arachnid) [inline EIPS/eip-1.md]: If you pronounce EIP as "eep" or "ee eye pee", "an" is the correct word.

Comment (guanqun) [inline EIPS/eip-1.md]: I thought so too, but everywhere else is using "a EIP" ...  Consistency is better.

Comment (cdetrio) [inline EIPS/eip-1.md]: Let's make it consistent and change them all to "an EIP".

Comment (guanqun) [inline EIPS/eip-1.md]: Done. changed all to "an EIP"

**Pull request merged**